products/healthcare/api.yaml

```diff
@@ -304,6 +304,61 @@ objects:
+          resource mutation is streamed to the new location in addition to the existing ones. When a location is removed
+          from the list, the server stops streaming to that location. Before adding a new config, you must add the required
+          bigquery.dataEditor role to your project's Cloud Healthcare Service Agent service account. Some lag (typically on
+          the order of dozens of seconds) is expected before the results show up in the streaming destination.
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::Array
+              name: 'resourceTypes'
+              description: |
+                Supply a FHIR resource type (such as "Patient" or "Observation"). See 
+                https://www.hl7.org/fhir/valueset-resource-types.html for a list of all FHIR resource types. The server treats
+                an empty list as an intent to stream all the supported resource types in this FHIR store.
+              item_type: Api::Type::String
+            - !ruby/object:Api::Type::NestedObject
+              name: bigqueryDestination
+              required: true
+              description: |
+                The destination BigQuery structure that contains both the dataset location and corresponding schema config.
+                The output is organized in one table per resource type. The server reuses the existing tables (if any) that
+                are named after the resource types, e.g. "Patient", "Observation". When there is no existing table for a given
+                resource type, the server attempts to create one.
+                See the [streaming config reference](https://cloud.google.com/healthcare/docs/reference/rest/v1beta1/projects.locations.datasets.fhirStores#streamconfig) for more details.
+              properties:
+                - !ruby/object:Api::Type::String
+                  name: datasetUri
+                  required: true
+                  description: |
+                    BigQuery URI to a dataset, up to 2000 characters long, in the format bq://projectId.bqDatasetId
+                - !ruby/object:Api::Type::NestedObject
+                  name: schemaConfig
+                  required: true
+                  description: |
+                    The configuration for the exported BigQuery schema.
+                  properties:
+                    - !ruby/object:Api::Type::Enum
+                      name: schemaType
+                      description: |
+                        Specifies the output schema type. Only ANALYTICS is supported at this time.
+                         * ANALYTICS: Analytics schema defined by the FHIR community.
+                          See https://github.com/FHIR/sql-on-fhir/blob/master/sql-on-fhir.md.
+                      default_value: :ANALYTICS
+                      values:
+                        - :ANALYTICS
```

Comment:
Only one value available as of now, but it looks like at least one more will be added eventually. The beta version of the api allows you to specify another value, but doing so returns an error stating it isn't supported.